### PR TITLE
DM-29704: Building cycle 0019. Update xml version to 9.0.0. Update ve…

### PR DESCRIPTION
**Pull request by wvreeven**
…rsions of required packages.

### CODE REVIEW

File: cycle/cycle.env [hunk 1]
```diff
@@ -36,50 +36,50 @@ stack=w_2021_12
 # OpenSplice setup
```
Comment (tribeiro) [inline cycle/cycle.env:36]: Please update lsstsqre and stack versions to:

```
lsstsqre=7-stack-lsst_distrib-w_2021_16
stack=w_2021_16
```

**Review by tribeiro — CHANGES_REQUESTED** [review]
See inline comment.

**Review by tribeiro — COMMENTED** [review]
Can you try to organize the commit history a bit better? I'd try to fold all changes to cycle.env into a single commit, then, one commit for each patch to the build scripts. 

Also, can you please add this to `build/common/lsst-ts-nexus.repo`?

```
[lsst-ts-daily]
name=LSST Telescope and Site - $basearch - Daily
baseurl=https://repo-nexus.lsst.org/nexus/repository/ts_yum/daily
failovermethod=priority
enabled=0
gpgcheck=0

[lsst-ts-bleed]
name=LSST Telescope and Site - $basearch - Bleed
baseurl=https://repo-nexus.lsst.org/nexus/repository/ts_yum/bleed
failovermethod=priority
enabled=0
gpgcheck=0
```

File: doc/user-guide/user-guide.rst [hunk 2]
```diff
@@ -211,9 +210,8 @@ This image uses the ``deploy_lsstsqre`` as a base image and uses ``eups`` to ins
 
 Because both ``Scheduler`` and ``MTAOS`` depends on ``lsst_sims`` they are built from a different base image than the base images above.
 We then proceed to build the ``lsst_sims`` image, which will install ``lsst_sims`` on ``deploy_lsstsqre``.
-Once this step is done, it is also possible to build ``aos_aoclc`` which is another intermediate stage in building ``MTAOS``.
 
-With ``lsst_sims`` and ``aos_aoclc`` built, it is then possible to build ``scheduler`` and ``mtaos`` in any order.
+With ``lsst_sims`` built, it is then possible to build ``scheduler`` and ``mtaos`` in any order.
```
Comment (tribeiro) [inline doc/user-guide/user-guide.rst:214]: We don't need sims to run mtaos anymore. So mtaos can be build as soon as `deploy_lsstsqre` is done.

File: doc/version-history.rst [hunk 3]
```diff
@@ -8,6 +8,12 @@ Version History
 .. No new work should be required in order to complete this section.
 .. Below is an example of a version history format.
 
+Cycle 0019
+==========
+
+* Release of XML 9.0.0
+* Updated the Dockerfiles of ptg and scriptqueue
```
Comment (tribeiro) [inline doc/version-history.rst:15]: Can you describe all the other changes you made as well? 

* Removed unnecessary container `aos_aoclc`.
* Updated conda-builder for arch64
* Update deploy-env for arch64

**Review by tribeiro — APPROVED** [review]
Looks good, I left two minor comments for you to consider before merging.

File: doc/user-guide/user-guide.rst (hunk 2 above)
Comment (wvreeven) [reply doc/user-guide/user-guide.rst]: Done

File: doc/version-history.rst (hunk 3 above)
Comment (wvreeven) [reply doc/version-history.rst:15]: Done

**Pull request merged**